### DISCUSSION TITLE
Export / download sorted data from a collection

### DISCUSSION THREAD

**Discussion by MAXPOL** [directus/directus#2818] [Archive]
Hello. 

I am created collection and add fields and add data, use filtering options made a selection. I want download / export filtering data in file (example EXCEL document). I can do it use graphic interfaces or i should use API or need write plugin? 
Thanks.

P.S. If i bad explained, i can add screenshots or describe more information.

**Comment by MAXPOL** [comment]
Dear experts, please tell me about my question.  Is it not possible to do this through the GUI?

**Comment by rijkvanzanten** [comment]
There's no export/import option in Directus yet, see https://github.com/directus/app/issues/711

**Reply by benhaynes** [reply]
We now have Data Export: https://docs.directus.io/guides/items/#exporting-items

Import is still in work.

**Comment by adrchen** [comment]
Considering that the answer was from last year, has this changed @rijkvanzanten ? Directus' front page mentions under **Connectivity and Logic** that it has a **Data Export** feature?

I'm using version 8.0, by the way.

**Reply by benhaynes** [reply]
First off, Directus 8 doesn't apply to this repo, you'll want this legacy one: https://github.com/directus/v8-archive

Second, we do have Data Export in Directus 9... Rijk's response was from before v9 was released. 

https://docs.directus.io/guides/items/#exporting-items
